InitializeRepo/readme.md

```diff
@@ -0,0 +1,48 @@
+# initialize.sh
+
+# Initializes director
+
+Prepends project filenames with repo name. All *.prjpcb, *.schdoc, *.pcbdoc.
```

Comment:
Do you mean that it replaces the base file name with the repo name?

Reply:
No, it adds the repo name to the beginning of the base name. New name = repo_base